### PR TITLE
Default compilation other classifiers

### LINKED ISSUE #709: Extend default compilation to other classifier models
In #695 we added default compilation for `BertClassifier` and `RobertaClassifier` models. Extend this to the other classifier tasks currently in our library:
* [ ] `DistilBertClassifier`
* [ ] `DebertaV3Classifier`
* [ ] `AlbertClassifier`
* [ ] `FNetClassifier`
* [ ] `XLMRobertaClassifier`

For now, we will use `Adam` with a constant learning rate of `5e-5`:
```python
# Default compilation
self.compile(
    loss=keras.losses.SparseCategoricalCrossentropy(from_logits=True),
    optimizer=keras.optimizers.Adam(5e-5),
    metrics=keras.metrics.SparseCategoricalAccuracy(),
    jit_compile=is_xla_compatible(self),
)
```

### PR DESCRIPTION
**Pull request by Plutone11011**
Resolve #709

Second commit is due to reformatting

### CODE REVIEW

File: keras_nlp/layers/masked_lm_mask_generator.py [hunk 1]
```diff
@@ -147,11 +147,7 @@ def call(self, inputs):
             # convert dense to ragged.
             inputs = tf.RaggedTensor.from_tensor(inputs)
 
-        (
-            token_ids,
-            mask_positions,
-            mask_ids,
-        ) = tf_text.mask_language_model(
+        (token_ids, mask_positions, mask_ids,) = tf_text.mask_language_model(
```
Comment (chenmoneygithub) [inline keras_nlp/layers/masked_lm_mask_generator.py:150]: This formatting is due to a black version change. Could you upgrade black and rerun the formatting? Thanks!

**Review by chenmoneygithub — COMMENTED** [review]
Thank you for the PR! 

PR looks good! there is a formatting issue caused by black version change. Please see the comment above.

**Review by mattdangerw — APPROVED** [review]
lgtm!

**Review by jbischof — APPROVED** [review]
Looks amazing. Welcome to KerasNLP @Plutone11011!
